lib/entry.js

```diff
@@ -75,8 +72,10 @@ Ep.addGetters = function (getters) {
     if (typeof getter === "function" &&
         // Ignore any requests for the exports.__esModule property."
         name !== "__esModule") {
-      // Should this throw if hasOwn.call(this.getters, name)?
-      entry.getters[name] = getter;
+      if (hasOwn.call(this.getters, name)) {
+        throw new SyntaxError("Identifier '" + name + "' exported more than once");
+      }
```

Comment:
If we could have this error thrown only during tests, and otherwise let the first getter win, I think that would be better than always throwing, since this is a runtime exception enforcing a compile-time guarantee.